### DISCUSSION TITLE
Carte hiver

### DISCUSSION THREAD

**Discussion by StrangerIM** [Crownicles/Crownicles#2587] [Nouvelles idées]
Petite idée comme ça qui m'est venue comme ça en regardant par la fenêtre 🌧️

Pourquoi ne pas créer une map du jeu version hiver ( càd avec de la neige et de la glace sur les plans d'eaux partout sauf dans le désert ) qui s'afficherait à la place de la map normale du 21 décembre au 21 mars ?

**Comment by StrangerIM** [comment]
Ça ajouterait un ptit plus au jeu qui est censé être en temps réel

**Comment by Ravenclaw-l** [comment]
Avec des easter eggs adaptés, genre le père noël.

**Comment by BastLast** [comment]
Pr info refaire la map et tout c'est environ 5 6 h de travail mega chiant de link des images / upload / mise en place des curseurs etc et c'est sans compter le temps de faire la carte elle meme. Perso j'ai grave la flemme

**Reply by StrangerIM** [reply]
Ok laisse tomber alors

**Reply by symsym-1629** [reply]
puis surtout on a plus l'abo inkarnate

**Comment by ghost** [comment]
ca pourra etre bien une nouvelle carte
